app/models/shop.rb

```diff
@@ -0,0 +1,10 @@
+
+class Shop < ApplicationRecord
+
+ 	validates :name, presence: true
+	validates :address, presence: true
+	validates :city, presence: true
+	validates :state, presence: true
+	validates :zip, presence: true
+	
+end
```

Comment:
You need unit tests for this new model (specifically validation test as per the test lesson in week 5